### ISSUE TITLE
pytorch sniper

### ISSUE THREAD

**Issue by liyangliu** [mahyarnajibi/SNIPER#116]
Hi, @bharatsingh430 @mahyarnajibi

I am trying to reproduce sniper under PyTorch, I have checked that my chip is cropped out similar to yours, and my training loss is under proper value. However when I use the trained model to do visualization, there will be a lot of part rois to be classified as objects. Could you please give me some suggestions? Thank you.

![image](https://user-images.githubusercontent.com/5159489/55667108-bceeab80-588a-11e9-8f8c-6a71e2795991.png)
![image](https://user-images.githubusercontent.com/5159489/55667117-cf68e500-588a-11e9-95ac-d56860f37973.png)
![image](https://user-images.githubusercontent.com/5159489/55667125-e0195b00-588a-11e9-8de4-e66196bfdb71.png)
![image](https://user-images.githubusercontent.com/5159489/55667138-06d79180-588b-11e9-9a1f-13f8edb88070.png)

BTW, I have successfully trained your mxnet model successfully on the same dataset (VOC2007), your results are visually better in terms of suppression of false positives of parts.
![image](https://user-images.githubusercontent.com/5159489/55667202-c9273880-588b-11e9-9d31-0101318cd973.png)

**Comment by musicshmily** [comment]
您好 ，pytorch版的 能给个连接吗

**Comment by mehrdadazizi72** [comment]
@liyangliu can you put your code on github or give access to test your code?

**Comment by zoe218** [comment]
@liyangliu hello, I'm trying to reproduce the sniper under tensorflow, and I meet the same problem with you. Have you found what's the problem? 
Thanks a lot~
